oxygen-account/src/main/java/com/oxygenxml/account/messages/TokenClaims.java

```diff
@@ -0,0 +1,35 @@
+package com.oxygenxml.account.messages;
+
+import lombok.Getter;
+
+/**
+ * Defines constants for JWT token claims used in the application.
+ */
+@Getter
+public enum TokenClaims {
+
+
+	/**
+	 * The claim representing the user ID.
+	 */
+	USER_ID("userId"),
+
+	/**
+	 * The claim representing the account creation date.
+	 */
+	CREATION_DATE("creationDate");
+
+	/**
+	 * The string key used to store the claim in the JWT.
+	 */
+	private final String tokenClaims;
```

Comment:
Numele fieldului replica numele clasei. Ar trebui sa denote numele unui atribut al unei instante a enumului curent. As redenumi field-ul in `name`